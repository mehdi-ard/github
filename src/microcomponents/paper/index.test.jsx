import React from 'react';
import renderer from 'react-test-renderer';
import Paper from '.';

it('renders with elevation', () => {
  expect(renderer.create(<Paper elevation={1}>dcdc</Paper>).toJSON()).toMatchSnapshot();
});
