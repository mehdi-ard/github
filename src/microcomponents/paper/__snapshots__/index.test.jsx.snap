// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with elevation 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation1 Paper-root-1  MuiPaper-rounded"
>
  dcdc
</div>
`;
